src/envoy/auth/jwt.cc

```diff
@@ -31,6 +31,7 @@
 namespace Envoy {
 namespace Http {
 namespace Auth {
+namespace {
 
 namespace Base64url {
```

Comment:
It seems that Base64url namespace is inside anonymous namespace.   Does it work?

I thought the anonymous namespace should the last one.